grai-integrations/source-bigquery/src/grai_source_bigquery/base.py

```diff
@@ -1,14 +1,18 @@
-from typing import List, Literal, Optional, Tuple
+from typing import List, Literal, Optional, Tuple, Union
 
 from grai_client.endpoints.client import BaseClient
 from grai_client.update import update
 from grai_schemas.base import Edge, Node
 
 from grai_source_bigquery.adapters import adapt_to_client
-from grai_source_bigquery.loader import BigqueryConnector
+from grai_source_bigquery.loader import BigqueryConnector, LoggingConnector
 
 
-def get_nodes_and_edges(connector: BigqueryConnector, version: Literal["v1"]) -> Tuple[List[Node], List[Edge]]:
+def get_nodes_and_edges(
+    connector: BigqueryConnector,
+    logging_connector: LoggingConnector = None,
+    version: Literal["v1"] = "v1",
+) -> Tuple[List[Node], List[Edge]]:
```

Comment:
I'm hesitant to change the API on `get_nodes_and_edges` so we call this function in quite a few places in the codebase. We've talked about standardizing this interface in the past so it may not matter. 

What if the LoggingConnector was instead an instance of a BigqueryConnector which exposed it's own `get_nodes`, `get_edges`, `get_nodes_and_edges`. Then the interface here wouldn't have to change.